### PR TITLE
Add new test 'Guest checkout: Use different invoice address'

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Add new test 'Guest checkout: Use different invoice address'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | `TEST_PATH="functional/FO/11_checkout/02_addresses/02_useDifferentAddressForInvoice.js" URL_FO=shopURL npm run specific-test`
| Possible impacts? | none

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24375)

### CODE REVIEW

**Review by nesrineabdmouleh — DISMISSED** [review]

**Review by NeOMakinG — DISMISSED** [review]

File: tests/UI/campaigns/functional/FO/11_checkout/02_addresses/02_useDifferentAddressForInvoice.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  describe('Make an oder with 2 different addresses for delivery and invoice', async () => {
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/11_checkout/02_addresses/02_useDifferentAddressForInvoice.js]:
```suggestion
  describe('Make an order with 2 differents addresses for delivery and invoice', async () => {
```

File: tests/UI/campaigns/functional/FO/11_checkout/02_addresses/02_useDifferentAddressForInvoice.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    });
+
+    it('should fill customer information', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'fillCustomerInformation', baseContext);
+
+      const isStepCompleted = await foCheckoutPage.setGuestPersonalInformation(page, guestData);
+      await expect(isStepCompleted).to.be.true;
+    });
+
+    it('should fill different delivery and invoice addresses address', async function () {
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/11_checkout/02_addresses/02_useDifferentAddressForInvoice.js]:
```suggestion
    it('should fill different delivery and invoice addresses', async function () {
```

**Review by Progi1984 — COMMENTED** [review]
Some feedbacks

File: tests/UI/campaigns/functional/FO/11_checkout/02_addresses/02_useDifferentAddressForInvoice.js (hunk 1 above)
Comment (atomiix) [inline tests/UI/campaigns/functional/FO/11_checkout/02_addresses/02_useDifferentAddressForInvoice.js]:
```suggestion
  describe('Make an order with 2 different addresses for delivery and invoice', async () => {
```

Comment (boubkerbribri) [reply tests/UI/campaigns/functional/FO/11_checkout/02_addresses/02_useDifferentAddressForInvoice.js]: First one is done. I don't think the second one is correct.

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @boubkerbribri

**Comment by Mwaqas1** [comment]
I am facing this issue, and i see you were working on this. is there any fix for this... using PS 1.7.8

Regards

**Pull request merged**